prow-configurations/prow-secrets-management.md

```diff
@@ -0,0 +1,76 @@
+# Prow secrets management
+
+## Overview
+
+Some jobs require using sensitive data. You need to encrypt data using KMS and store them in GCS.
+This document shows the commands necessary to create a service account and store its encrypted key in GCS bucket. This document assumes that you are logged in to the Google Cloud project with administrative rights.
+
+## Prerequisites
+
+ - [gcloud](https://cloud.google.com/sdk/gcloud/) 
+ - basic knowledge about [GCP key rings and keys](https://cloud.google.com/kms/docs/creating-keys).
+
+For your convenience, export these data:
+ - PROJECT_NAME - Google Cloud Project
+ - BUCKET_NAME - GCS bucket in the $PROJECT_NAME where Prow secrets are stored
+ - KEYRING_NAME - KMS key ring
+ - ENCRYPTION_KEY_NAME - key in the $KEYRING_NAME, used for data encryption
+
+## Secrets management
+
+>**NOTE:** Before following this guide, check Prow secrets setup for the $PROJECT_NAME.
```

Comment:
- Before following -> Before you follow
- for the $PROJECT_NAME. -> for the Google Cloud project.